cloud/ansible/full-deployment.yaml

```diff
@@ -0,0 +1,108 @@
+- name: Test connection
+  connection: ansible.netcommon.network_cli
+  gather_facts: false
+  hosts: public-machine
+  tasks:
+    - name: get pwd
+      command: pwd
+      register: result
+
+    - name: Display the config
+      debug:
+        msg: "The hostname is {{ inventory_hostname}} and the pwd is {{ result.stdout }}"
+
+- name: Install docker
+  hosts: public-machine
+  become: yes
+  vars:
+    pkgstoinstall: [ libffi-dev, libssl-dev, python3, python3-pip ]
+  tasks:
+    - name: Install a list of packages
+      apt:
+        name: "{{ pkgstoinstall }}"
+        update_cache: yes
+
+    - name: Remove python-configparser package
+      apt:
+        name: python-configparser
+        state: absent
+
+    - name: get docker convenience script
+      shell: curl -fsSL https://get.docker.com -o get-docker.sh
+      args:
+        creates: /home/azureuser/get-docker.sh
+
+    - name: install docker
+      shell: sh /home/azureuser/get-docker.sh
+      args:
+        creates: /usr/bin/docker
+
+- name: Create users and add SSH keys
+  hosts: public-machine
+  become: yes
+  vars:
+    users:
+      - username: mgarbowski
+        full_name: Mikołaj Garbowski
```

Comment:
jesli dobrze rozumiem, to tu i w users.yaml powtarza sie ta logika? tak chyba nie powinno byc?

Reply:
full deployment to jest kompilacja wszystkich pozostałych plików. Mogę je usunąć ale zostawiłem żeby można było granularnie te rzeczy odpalać też

Reply:
aa spoko kumam

Reply:
https://serverfault.com/questions/750856/how-to-run-multiple-playbooks-in-order-with-ansible

Reply:
bedzie w ten sposob jedno zrodlo prawdy

Reply:
dobrze gada @mbienkowsk nie duplikujmy tego

Reply:
usunąłem te pojedyncze, teraz jest jeden konkretny który robi deployment jak należy